src/api/participants/routes.js

```diff
@@ -311,7 +311,7 @@ module.exports = [
       description: 'Record Funds In or Out of participant account',
       validate: {
         payload: Joi.object({
-          transferId: Joi.string().pattern(/^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-7][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]}$|^[0-9A-HJKMNP-TV-Z]{26}$6})$/).required(),
+          transferId: Joi.string().pattern(/^(?:[0-9a-f]{8}-[0-9a-f]{4}-[1-7][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$|^[0-9A-HJKMNP-TV-Z]{26})$/).required(),
```

Comment:
why not use `transferId: Joi.string().guid().required()` here also?

Reply:
Not sure. :thinking: I'll address this in a follow up.

Reply:
Nvm. Added in commit.